### PR TITLE
Add Release Note for KubeDB v2024.3.16

### PR DESCRIPTION
**Pull request by raihankhan**

### CODE REVIEW

File: content/post/kubedb-v2024.03.16/index.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+      maxAllowed:
+        cpu: 2
+        memory: 3Gi
+      controlledResources: ["cpu", "memory"]
+```
+
+#### New Version Supports
+In this release, we have extended support to include two new versions each for both Kafka and Kafka Connect Cluster.
+- `3.6.1`
+- `3.5.2`
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.03.16/index.md:168]: Need a new line.

**Review by MobarakHsn — APPROVED** [review]
LGTM

File: content/post/kubedb-v2024.03.16/index.md (hunk 1 above)
Comment (raihankhan) [reply content/post/kubedb-v2024.03.16/index.md:168]: removed

File: content/post/kubedb-v2024.03.16/index.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+Note `walKeepSegment`, `walKeepSize`, `maxSlotWALKeepSize` resembles `wal_keep_segment`, `wal_keep_size` and `max_slot_wal_keep_size` of `postgresql.conf` file.
+
+
+## Zookeeper
+
+This release includes Grafana dashboards for easier monitoring of KubeDB managed ZooKeeper. The grafana dashboard shows several ZooKeeper specific data, status and diagram of memory and cpu consumption. You can check the dashboard to see the overall health of your zookeeper servers easily. As usual KubeDB provided Grafana dashboards comes in a bundle of three - Summary dashboard for overall monitoring of the database cluster, Database dashboard for database insights and Pod dashboard for monitoring database pods specifically. Here's a preview of the summary dashboard for Zookeeper.
+
+![zookeeper-dashboard](images/kubedb-zookeeper-summary.png)
+
+A step-by-step guide to monitoring is given [here](https://github.com/appscode/grafana-dashboards/tree/master/zookeeper)
```
Comment (AbdullahAlShaad) [inline content/post/kubedb-v2024.03.16/index.md:94]: Add fullstop after this line.

**Review by AbdullahAlShaad — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2024.03.16/index.md [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+
+## MariaDB
+
+### Archiver
+
+KubeDB now supports continuous archiving of a MariaDB database. So, you can also do point-in-time recovery (PITR) restoration of the database at any point.
+
+To use this feature, You need KubeStash installed in your cluster. KubeStash (aka Stash 2.0) is a ground up rewrite of Stash with various improvements planned. It works with any existing KubeDB or Stash license key. 
+
+To use the continuous archiving feature, we have introduced a CRD on the KubeDB side, named MariaDBArchiver. Here are all the details of using MariaDBArchiver . In short, you need to create the following resources:
```
Comment (hmsayem) [inline content/post/kubedb-v2024.03.16/index.md:228]:
```suggestion
To use the continuous archiving feature, we have introduced a CRD on the KubeDB side, named `MariaDBArchiver`. Here are all the details of using `MariaDBArchiver` . In short, you need to create the following resources:
```

File: content/post/kubedb-v2024.03.16/index.md [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+   ref:
+     name: s3-storage
+     namespace: demo
+   subDir: test
+```
+
+Now After creating this archiver CR, if we create a MariaDB with archiver: **"true"** label, in the same namespace (as per the double-optin configured in .spec.databases field), The KubeDB operator will start doing the following tasks:
+
+Create a BackupConfiguration named **<db-name>-backup**
+Start syncing mysql wal files to the directory **<sub-directory>/<database_namespace>/<database_name>/binlog-backup**
```
Comment (hmsayem) [inline content/post/kubedb-v2024.03.16/index.md:289]: should be bullet points.

File: content/post/kubedb-v2024.03.16/index.md [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+
+When the BackupConfiguration is created KubeStash operator will start doing the following tasks:
+Creates two Repositories with convention **<db-name>-full** and **<db-name>-manifest**.
+
+Takes full backup every day at 3:30 (`.spec.fullBackup.scheduler`) to **<db-name>-full** repository and takes manifest backup every day at 3:30 (`.spec.manifestBackup.scheduler`) to **<db-name>-manifest** repository.
```
Comment (hmsayem) [inline content/post/kubedb-v2024.03.16/index.md:294]: should be bullet points.

**Review by hmsayem — CHANGES_REQUESTED** [review]

**Review by obaydullahmhs — APPROVED** [review]
LGTM!

**Review by AbdullahAlShaad — APPROVED** [review]
LGTM!

**Comment by hmsayem** [comment]
LGTM

**Review by hmsayem — DISMISSED** [review]

**Review by ArnobKumarSaha — DISMISSED** [review]

**Pull request merged**